docs/source/API/core/view/view_alloc.rst

```diff
@@ -0,0 +1,54 @@
+``view_alloc()``
+================
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Header File: ``<Kokkos_View.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+
+    Kokkos::view_alloc(exec_space, Kokkos::WithoutInitializing, "ViewString");
+    Kokkos::view_wrap(pointer_to_wrapping_memory);
+
+Create View allocation parameter bundle from argument list. Valid argument list members are:
+
+* label as ``C``-string or ``std::string``
+* memory space instance of the ``View::memory_space`` type
+* execution space instance able to access ``View::memory_space``
+* ``Kokkos::WithoutInitializing`` to bypass initialization
+* ``Kokkos::AllowPadding`` to allow allocation to pad dimensions for memory alignment
+* a pointer to create an unmanaged View wrapping that pointer
+
+..
+   Synopsis
+   --------
+
+   .. code-block:: cpp
+
+       template <class... Args>
+       /*implementation-detail*/
+       view_alloc(Args const&... args);
+
+       template <class... Args>
+       KOKKOS_FUNCTION
+       /*implementation-detail*/
+       view_wrap(Args const&... args);
```

Comment:
I am confused why is there a `Synopsis` but not in the screenshot

Reply:
we are removing the synopsis where it is redundant